astroquery/solarsystem/pds/core.py

```diff
@@ -0,0 +1,368 @@
+                        )
+                    elif "Observer-planet distance (km)" in l[0]:
+                        systemtable["d_obs"] = (
+                            float(l[1].split("x")[0].strip(", \n")) * 1e6 * u.km
+                        )
+                    elif "Light travel time" in l[0]:
+                        systemtable["light_time"] = float(l[1].strip(", \n")) * u.second
+                    else:
+                        pass
+
+            # --------- below this line, planet-specific info ------------
+            # Uranus individual rings data
+            elif group.startswith("Ring    "):
+                ringtable = table.QTable.read("     " + group, format="ascii.fixed_width",
+                    col_starts=(5, 18, 29),
+                    col_ends=(18, 29, 36),
+                    names=("ring", "pericenter", "ascending node"),
+                    units=([None, u.deg, u.deg]))
+
+            # Saturn F-ring data
+            elif group.startswith("F Ring"):
+                lines = group.split("\n")
+                for line in lines:
+                    l = line.split(":")
+                    if "F Ring pericenter" in l[0]:
+                        peri = float(re.sub("[a-zA-Z]+", "", l[1]).strip(", \n()"))
+                    elif "F Ring ascending node" in l[0]:
+                        ascn = float(l[1].strip(", \n"))
+                ringtable = table.QTable(
+                    [["F"], [peri], [ascn]],
+                    names=("ring", "pericenter", "ascending node"),
+                    units=(None, u.deg, u.deg),
+                )
+
+            # Neptune ring arcs data
+            elif group.startswith("Courage"):
+                lines = group.split("\n")
+                for i in range(len(lines)):
+                    l = lines[i].split(":")
+                    ring = l[0].split("longitude")[0].strip(", \n")
+                    [min_angle, max_angle] = [
+                        float(s.strip(", \n"))
+                        for s in re.sub("[a-zA-Z]+", "", l[1]).strip(", \n()").split()
+                    ]
+                    if i == 0:
+                        ringtable = table.QTable(
+                            [[ring], [min_angle], [max_angle]],
+                            names=("ring", "min_angle", "max_angle"),
+                            units=(None, u.deg, u.deg),
+                        )
+                    else:
+                        ringtable.add_row([ring, min_angle*u.deg, max_angle*u.deg])
+
+            else:
+                pass
+
+        # do some cleanup from the parsing job
+        if ringtable is not None:
+            ringtable.add_index("ring")
+
+        bodytable = table.join(bodytable, bodytable2)  # concatenate minor body table
+        bodytable.add_index("Body")
+
+        systemtable["epoch"] = Time(epoch, format="iso", scale="utc")  # add obs time to systemtable
+
+        return systemtable, bodytable, ringtable
```

Comment:
Would it be a better idea to make the information in `systemtable` available in the metadata of `bodytable` and `ringtable` instead of returning it separately?

Reply:
That's a good idea.